_tickers.md

```diff
@@ -451,3 +451,41 @@ This method allows developers to find all exchanges rates relative to a given cu
 
 ### Response
 Returns an associative array containing the current rates Uphold has on record for the currency specified. If no currency is specified on the endpoint, USD currency pair will be returned by default.
+
+## Get Tickers for currency pair
+
+```bash
+curl https://api.uphold.com/v0/ticker/USDEUR
+```
+
+> The above command returns JSON in the following format:
+
+```json
+{
+  "ask":"0.87742",
+  "bid":"0.87742",
+  "currency":"EUR"
+}
+```
+
+```bash
+curl https://api.uphold.com/v0/ticker/EURUSD
+```
+
+> The above command returns JSON in the following format:
+
+```json
+{
+  "ask":"1.13985",
+  "bid":"1.13985",
+  "currency":"USD"
+}
+```
+
+This method allows developers to find the exchange rate of a currency relative to any other currency.  The order of the currencies in the pair affects the output. As is shown in the examples, the endpoint will provide the exchange rate of the first currency to the second currency.
```

Comment:
```suggestion
This method allows developers to find the exchange rate of a currency relative to any other currency.  The order of the currencies in the pair affects the output. As is shown in the example, the endpoint will provide the exchange rate from the first currency to the second currency.
```

Reply:
The text "The order of the currencies in the pair affects the output" should be placed in a notice box (`<aside class="notice">`), like [here](https://github.com/uphold/docs/pull/108/files) for example.

Reply:
You addressed the second comment but not the first one (with the diff). It's a bit hard to read as a single long line (which is why I prefer hard-wrapped text, but that a topic for another day ;)), but the suggestions still stand.

Reply:
Now it's done! :)